commons-rest-test/src/main/java/com/mule/connectors/commons/rest/test/assertion/raml/RamlResponseAdapter.java

```diff
@@ -46,6 +47,6 @@ public String getContentType() {
 
     @Override
     public byte[] getContent() {
-        return SerializationUtils.serialize(content);
+        return content.getBytes();
```

Comment:
Use UTF-8 Charset here and the getBytes(Charset) implementation (it doesn't throw an exception).
This use depends on the default system encoder and if it were to change it'll be trouble.